.gitlab/deploy_containers/conditions.yml

```diff
@@ -0,0 +1,80 @@
+
+
+.manual_auto_on_rc:
```

Comment:
Could you add a comment to clarify the intent of this rule?

Reply:
Added some comments